sync-bintray.sh

```diff
@@ -70,26 +66,27 @@ fi
 
 if (( ${#files[@]} > 0 ))
 then
-  # NB: Archives sent to bintray for exploding must not have directory entries inside, just the
```

Comment:
NB: frustrating problems with rate limits led to the zip change a ways back, and I use FILTER above today to do incremental syncs.  Not objecting to the change, but there is a resaon for the zip.  Should not be a problem though since I expect most syncs will be of ~10 files which is under the daily rate limit.